### PR TITLE
[ADD][account_invoice_period_usability] module to display fiscal period next to supplier invoice date

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**

### CODE REVIEW

File: account_invoice_period_usability/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ],
+    'data': [
+        'views/account_invoice_view.xml',
+    ],
+    'qweb': [
+    ],
+    'demo': [
+    ],
+    'test': [
+    ],
```
Comment (adrienpeiffer) [inline account_invoice_period_usability/__openerp__.py]: Do you think that empty entries are necessary?

**Comment by adrienpeiffer** [comment]
A little remark in manifest, otherwise :+1:

File: account_invoice_period_usability/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Display in the invoice supplier form the fiscal period next to the invoice date
```
Comment (adrienpeiffer) [inline account_invoice_period_usability/README.md]: Can you add the licence badge at the beginning ?

.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License

**Comment by adrienpeiffer** [comment]
A little remark in manifest and in readme, otherwise :+1:

File: account_invoice_period_usability/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Account Invoice Period Usability',
+    'summary': """
+Display in the supplier invoice form the fiscal period
+next to the invoice date""",
+    'author': 'ACSONE SA/NV',
```
Comment (adrienpeiffer) [inline account_invoice_period_usability/__openerp__.py]: Can you add OCA as author like this : ACSONE SA/NV,Odoo Community Association (OCA)

File: account_invoice_period_usability/views/account_invoice_view.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            <field name="arch" type="xml">
+                <field name="period_id" position="replace"/>
+                <field name="date_invoice" position="replace">
+                    <label for="date_invoice"/>
+                    <div>
+                        <field name="date_invoice"
+                               on_change="onchange_payment_term_date_invoice(payment_term, date_invoice)"
+                               class="oe_inline"/>
+
+                        <field name="period_id" domain="[('state', '=', 'draft'), ('company_id', '=', company_id)]"
```
Comment (sbidoul) [inline account_invoice_period_usability/views/account_invoice_view.xml]: add ('special', '=', False) in the domain as we can't use opening periods here

**Comment by sbidoul** [comment]
I think the README has to be realigned with the up-to-date template (the bug tracker section is missing).

Otherwise :+1: (code review and functional test)

**Comment by pedrobaeza** [comment]
:+1: with the bug tracker remark

**Comment by legalsylvain** [comment]
@Olivier-LAURENT : Could you make minor changes related to bug tracker), to merge this PR ?
kind regards.

**Comment by sbidoul** [comment]
@legalsylvain @pedrobaeza I improved the readme, so it should be good for merging.

**Comment by pedrobaeza** [comment]
:+1: for me

**Comment by legalsylvain** [comment]
Thanks.

**Pull request merged**